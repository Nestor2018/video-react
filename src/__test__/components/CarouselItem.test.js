import React from 'react';
import { shallow } from 'enzyme';
import { create } from 'react-test-renderer';

import CarouselItem from '../../components/CarouselItem';
import ProviderMock from '../../__mocks__/ProviderMock';

describe('<CarouseItem />', () => {
	const carouselItem = shallow(
		<ProviderMock>
			<CarouselItem />
		</ProviderMock>
	);
	test('Render del componente CarouselItem', () => {
		expect(carouselItem.length).toEqual(1);
	});
});

describe('CarouselItem Snapshot', () => {
	const carouseItem = create(
		<ProviderMock>
			<CarouselItem />
		</ProviderMock>
	);
	test('Comprobar la UI del componente CarouselItem', () => {
		expect(carouseItem.toJSON()).toMatchSnapshot();
	});
});
