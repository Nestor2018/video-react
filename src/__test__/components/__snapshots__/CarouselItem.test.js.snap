// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CarouselItem Snapshot Comprobar la UI del componente CarouselItem 1`] = `
<div
  className="carousel-item"
>
  <img
    className="carousel-item__img"
  />
  <div
    className="carousel-item__details"
  >
    <div>
      <img
        alt="Play Icon"
        className="carousel-item__details--img"
        src="test-file-stub"
      />
      <img
        alt="Plus Icon"
        className="carousel-item__details--img"
        onClick={[Function]}
        src="test-file-stub"
      />
    </div>
    <p
      className="carousel-item__details--title"
    />
    <p
      className="carousel-item__details--subtitle"
    >
      undefined undefined undefined
    </p>
  </div>
</div>
`;
